client/src/components/Header/Header.js

```diff
@@ -34,97 +34,83 @@ export default class Header extends React.Component {
     }
 
     render() {
+        let headerLinks = this.links.map(link =>
+            <NavLink onClick={e => {
+                e.preventDefault();
+                window.location.replace(link.route);
+            }} to={link.route} exact={link.exact} className="header-link mr-4" activeClassName="active" key={link.name}>{link.name}</NavLink>
+        );
         return (
             <div className="header" id="pageHeader" ref={element => this.divRef = element}>
-                {
-                    this.state.screenWidth > 830 && <>
```

Comment:
Rather than having separate left/tabletLeft/right/tabletRight/etc, there is now one left/center/right div and css breakpoints are used to change styles depending on screen size. I'm still using some javascript screen size checking here when it comes to whether or not something should be rendered at all on that screen, but there's no copy and pasted markup anymore